gcloud/credentials.py

```diff
@@ -59,6 +68,113 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def _store_user_credential(credential):
+    """Stores a user credential as a well-known file.
+
+    Prompts user first if they want to store the minted token and
```

Comment:
I'm not sure? It's a term I've been using for a long time but I've implemented OAuth 2.0 at the library level like 3 or 4 times, so I am very biased.

It's like a factory spitting out or "minting" goods.